pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -377,6 +377,7 @@ func (p *PriorityQueue) backoffPod(pod *v1.Pod) {
 // p.receivedMoveRequest is false or to backoff queue if p.receivedMoveRequest
 // is true but pod is subject to backoff. In other cases, it adds pod to active
 // queue.
+// It also clears receivedMoveRequest to mark the beginning of a new scheduling cycle.
```

Comment:
I think you should remove "to mark the beginning..." from this comment. This is not the beginning of a new scheduling cycle.

Reply:
Done.